app/assets/json/translations/react/en.json

```diff
@@ -48,11 +48,22 @@
   "listings.lotteryResults.show": "Show Lottery Results",
   "listings.lotteryResults.subtitle": "You can now check lottery results on the remaining listings.",
   "listings.lotteryResults.title": "Lottery Results",
+  "listings.mobilityImpairments.hearing": "Hearing Impairments",
```

Comment:
minor: I wonder if we should call this `listings.prioritiesDescriptor.hearing` or similar, since they're not all mobility-related